### ISSUE TITLE
NCBI taxonomy file not found

### ISSUE THREAD

**Issue by oconneru** [DerrickWood/kraken2#246]
I'm trying to build a new library using the basic --download-taxonomy, but it isn't finding the nucl_est.accession2taxid.gz file. Indeed when I manually go to the NCBI ftp server, it isn't there. Is there a workaround?

**Comment by jenniferlu717** [comment]
I believe you're using an older kraken2 script? We fixed the --download-taxonomy file when NCBI changed their taxonomy structure. They only have the nucl_wgs and nucl_gb accession2taxid files. 

If you update your kraken code, it should work.

**Comment by margeti** [comment]
I work with kraken2 version 2.0.8_beta and have the same problem... 

Module for kraken2 version 2.0.8_beta loaded
Downloading nucleotide gb accession to taxon map...rsync: error while loading shared libraries: libiconv.so.2: cannot open shared object file: No such file or directory

then I realised there is a 2.0.9. version available. but when I tried to install it with conda it would only install the 2.0.7. version.... I guess with 0.7 I will have the same problem?

 +## Package Plan ##

  environment location: /home/ollie/lschulte/miniconda3

  added / updated specs:
    - kraken2


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    kraken2-2.0.7_beta         |  pl526h2d50403_0         132 KB  bioconda/label/cf201901
    ------------------------------------------------------------
                                           Total:         132 KB

The following NEW packages will be INSTALLED:

  kraken2            bioconda/label/cf201901/linux-64::kraken2-2.0.7_beta-pl526h2d50403_0


Proceed ([y]/n)? n
